violetplugin-classdiagram/src/main/java/com/horstmann/violet/product/diagram/classes/node/InterfaceNode.java

```diff
@@ -192,6 +203,7 @@ public void revertUpdate()
 
     private static final int MIN_NAME_HEIGHT = 45;
     private static final int MIN_WIDTH = 100;
+	private boolean VISIBLE_METHODS_AND_ATRIBUTES = true;
     private static final String STATIC = "<<static>>";
```

Comment:
Shoudln't it be the special characters « » ? If i have right, then use unicodes.